src/test/scala/chiselTests/VectorPacketIO.scala

```diff
@@ -0,0 +1,57 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import Chisel._
+import Chisel.testers.BasicTester
+
+/**
+  * This test used to fail when assignment statements were
+  * contained in DeqIO and EnqIO constructors.
+  * The symptom is creation of a firrtl file
+  * with missing declarations, the problem is exposed by
+  * the creation of the val outs in VectorPacketIO
+  * NOTE: The problem does not exist now because the initialization
+  * code has been removed from DeqIO and EnqIO
+  */
+class Packet extends Bundle {
```

Comment:
If the point of this is just to test EnqIO/DeqIO, perhaps a dedicated unit test would be clearer (rather than a complete design that just happens to test EnqIO/DeqIO)?

Also, if relying on behavior that may change in the future, I'd also advocate including that caveat in the test, since people may look at these as examples.